FormalConjectures/Wikipedia/Brocard.lean

```diff
@@ -0,0 +1,35 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Brocard's Conjecture
+
+For every n ≥ 2, between the squares of the n-th and (n+1)-th primes,
+there are at least four prime numbers.
+
+*Reference:* [Wikipedia](https://en.wikipedia.org/wiki/Brocard%27s_conjecture)
+-/
+
+@[category research open, AMS 11]
+theorem brocard_conjecture :
+  ∀ n : ℕ, n ≥ 2 →
```

Comment:
```suggestion
theorem brocard_conjecture (n : ℕ) (hn : n ≥ 1)
```

Reply:
Note: the wiki page is indexing primes from `1` while we're indexing them from `0` so we should also shift the condition on the index `n`.